tests/test_readme.py

```
@@ -0,0 +1,180 @@
+"""Test suite for README.md validation."""
+
+import os
+import re
+from pathlib import Path
+
+
+class TestREADMEStructure:
+    """Test README.md file structure and content."""
+    
+    def test_readme_file_exists(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_readme_file_exists`

Add return type annotation: `None`

(ANN201)